src/main/resources/db/migration/V1_65__elinks_alter_tables.sql

```diff
@@ -0,0 +1,40 @@
+--Alter dbjudicialdata.judicial_user_profile
+ALTER TABLE dbjudicialdata.judicial_user_profile ADD COLUMN deleted_flag boolean;
+ALTER TABLE dbjudicialdata.judicial_user_profile ADD COLUMN date_of_deletion TIMESTAMP;
+
+--Alter dbjudicialdata.judicial_office_appointment
+ALTER TABLE dbjudicialdata.judicial_office_appointment ALTER COLUMN base_location_id SET NOT NULL;
+ALTER TABLE dbjudicialdata.judicial_office_appointment ALTER COLUMN appointment SET NOT NULL;
+ALTER TABLE dbjudicialdata.judicial_office_appointment ALTER COLUMN contract_type_id SET NOT NULL;
+
+--Alter dbjudicialdata.judicial_office_authorisation
+ALTER TABLE dbjudicialdata.judicial_office_authorisation ALTER COLUMN jurisdiction_id SET NOT NULL;
+ALTER TABLE dbjudicialdata.judicial_office_authorisation ALTER COLUMN lower_level SET NOT NULL;
+ALTER TABLE dbjudicialdata.judicial_office_authorisation ALTER COLUMN ticket_code SET NOT NULL;
+
+--Alter dbjudicialdata.judicial_role_type
+ALTER TABLE dbjudicialdata.judicial_role_type ALTER COLUMN title SET NOT NULL;
+
+
+--Alter dbjudicialdata.judicial_office_appointment
+ALTER TABLE dbjudicialdata.judicial_office_appointment  DROP CONSTRAINT cft_region_id_fk1;
+ALTER TABLE dbjudicialdata.judicial_office_appointment RENAME COLUMN cft_region_id TO hmcts_region_id;
+ALTER TABLE dbjudicialdata.judicial_office_appointment ALTER COLUMN hmcts_region_id TYPE varchar(64);
+
+--create dbjudicialdata.hmcts_region_type
+CREATE TABLE dbjudicialdata.hmcts_region_type (
+	hmcts_region_id varchar(64) NOT NULL,
+	hmcts_region_desc_en varchar(256) NOT NULL,
+	hmcts_region_desc_cy varchar(256) NULL,
```

Comment:
Loke like NULL is the default value so you can remove it from line no 28

Reply:
Needful done @manukundloo-hmcts